### ISSUE TITLE
[Regression] google_play_track_version_codes returns an array of strings (instead of an array of integers)

### ISSUE THREAD

**Issue by edu-zamora** [fastlane/fastlane#15622]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description

I updated to 2.135.0 (from 2.124.0) and our lane for publishing the Android build fails because `google_play_track_version_codes` now returns an array of strings, instead of returning an array of integers (as specified in its documentation here: https://docs.fastlane.tools/actions/google_play_track_version_codes/).

##### Command executed
`fastlane publish`, which is a custom lane that ends up calling the next functions:

```
	def self.latest_build_number
		tracks = ["internal", "alpha", "beta", "rollout", "production"]
		version_codes = []
		tracks.each do |track|
			version_codes.concat(google_play_track_version_codes(track: track))
		end
		version_codes.max
	end

	def self.next_build_number
		latest_build_number + 1
	end
```

##### Complete output when running fastlane, including the stack trace and command used

<details>
  <pre>
$ yarn env:test && pushd android && fastlane publish && popd
$ cp env/.env.test .env && cp .env android/fastlane/ && cp .env ios/fastlane/
~/dev/fluent-forever/monorepo/app/android ~/dev/fluent-forever/monorepo/app
[✔] 🚀
[16:59:34]: fastlane detected a Gemfile in the current directory
[16:59:34]: however it seems like you don't use `bundle exec`
[16:59:34]: to launch fastlane faster, please use
[16:59:34]:
[16:59:34]: $ bundle exec fastlane publish
[16:59:34]:
[16:59:34]: Get started using a Gemfile for fastlane https://docs.fastlane.tools/getting-started/ios/setup/#use-a-gemfile
+------------------------------------+---------+-----------------------------------------------------------------------------+
|                                                        Used plugins                                                        |
+------------------------------------+---------+-----------------------------------------------------------------------------+
| Plugin                             | Version | Action                                                                      |
+------------------------------------+---------+-----------------------------------------------------------------------------+
| fastlane-plugin-versioning_android | 0.1.0   | android_get_version_name android_set_version_code android_set_version_name  |
|                                    |         | android_get_version_code                                                    |
+------------------------------------+---------+-----------------------------------------------------------------------------+

[16:59:35]: ------------------------------
[16:59:35]: --- Step: default_platform ---
[16:59:35]: ------------------------------
[16:59:35]: Driving the lane 'android publish' 🚀
[16:59:35]: ---------------------------------------------
[16:59:35]: --- Step: google_play_track_version_codes ---
[16:59:35]: ---------------------------------------------
[16:59:41]: Found '123' version codes in track 'internal'
[16:59:41]: ---------------------------------------------
[16:59:41]: --- Step: google_play_track_version_codes ---
[16:59:41]: ---------------------------------------------
[16:59:50]: Found '443' version codes in track 'alpha'
[16:59:50]: ---------------------------------------------
[16:59:50]: --- Step: google_play_track_version_codes ---
[16:59:50]: ---------------------------------------------
[16:59:52]: No version codes found in track 'beta'
[16:59:52]: ---------------------------------------------
[16:59:52]: --- Step: google_play_track_version_codes ---
[16:59:52]: ---------------------------------------------
[17:00:00]: No version codes found in track 'rollout'
[17:00:00]: ---------------------------------------------
[17:00:00]: --- Step: google_play_track_version_codes ---
[17:00:00]: ---------------------------------------------
[17:00:05]: Found '399' version codes in track 'production'
+------------------+-----------------+
|            Lane Context            |
+------------------+-----------------+
| DEFAULT_PLATFORM | android         |
| PLATFORM_NAME    | android         |
| LANE_NAME        | android publish |
+------------------+-----------------+
[17:00:05]: Error in your Fastfile at line 42
[17:00:05]:     40:
[17:00:05]:     41:		def self.next_build_number
[17:00:05]:  => 42:			latest_build_number + 1
[17:00:05]:     43:		end
[17:00:05]:     44:
[17:00:05]: no implicit conversion of Integer into String

+------+---------------------------------+-------------+
|                   fastlane summary                   |
+------+---------------------------------+-------------+
| Step | Action                          | Time (in s) |
+------+---------------------------------+-------------+
| 1    | default_platform                | 0           |
| 2    | google_play_track_version_codes | 6           |
| 3    | google_play_track_version_codes | 8           |
| 4    | google_play_track_version_codes | 1           |
| 5    | google_play_track_version_codes | 8           |
| 6    | google_play_track_version_codes | 4           |
+------+---------------------------------+-------------+

[17:00:05]: fastlane finished with errors

Looking for related GitHub issues on fastlane/fastlane...

➡️  Error: no implicit conversion of Integer into String
    https://github.com/fastlane/fastlane/issues/13799 [closed] 3 💬
    02 Feb 2019

➡️  [PEM] `fastlane pem` fails on multiple app_identifiers `no implicit conversion of Array into String`
    https://github.com/fastlane/fastlane/issues/13559 [closed] 15 💬
    06 Mar 2019

➡️  latest_testflight_build_number + 1 -> [!] no implicit conversion of Fixnum into String
    https://github.com/fastlane/fastlane/issues/12097 [closed] 11 💬
    23 May 2018

and 21 more at: https://github.com/fastlane/fastlane/search?q=no%20implicit%20conversion%20of%20Integer%20into%20String&type=Issues&utf8=✓

🔗  You can ⌘ + double-click on links to open them directly in your browser.
Traceback (most recent call last):
	20: from /usr/local/bin/fastlane:23:in `<main>'
	19: from /usr/local/bin/fastlane:23:in `load'
	18: from /usr/local/lib/ruby/gems/2.6.0/gems/fastlane-2.135.0/bin/fastlane:23:in `<top (required)>'
	17: from /usr/local/lib/ruby/gems/2.6.0/gems/fastlane-2.135.0/fastlane/lib/fastlane/cli_tools_distributor.rb:119:in `take_off'
	16: from /usr/local/lib/ruby/gems/2.6.0/gems/fastlane-2.135.0/fastlane/lib/fastlane/commands_generator.rb:41:in `start'
	15: from /usr/local/lib/ruby/gems/2.6.0/gems/fastlane-2.135.0/fastlane/lib/fastlane/commands_generator.rb:349:in `run'
	14: from /usr/local/lib/ruby/gems/2.6.0/gems/commander-fastlane-4.4.6/lib/commander/delegates.rb:15:in `run!'
	13: from /usr/local/lib/ruby/gems/2.6.0/gems/fastlane-2.135.0/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:76:in `run!'
	12: from /usr/local/lib/ruby/gems/2.6.0/gems/commander-fastlane-4.4.6/lib/commander/runner.rb:476:in `run_active_command'
	11: from /usr/local/lib/ruby/gems/2.6.0/gems/commander-fastlane-4.4.6/lib/commander/command.rb:153:in `run'
	10: from /usr/local/lib/ruby/gems/2.6.0/gems/commander-fastlane-4.4.6/lib/commander/command.rb:178:in `call'
	 9: from /usr/local/lib/ruby/gems/2.6.0/gems/fastlane-2.135.0/fastlane/lib/fastlane/commands_generator.rb:108:in `block (2 levels) in run'
	 8: from /usr/local/lib/ruby/gems/2.6.0/gems/fastlane-2.135.0/fastlane/lib/fastlane/command_line_handler.rb:36:in `handle'
	 7: from /usr/local/lib/ruby/gems/2.6.0/gems/fastlane-2.135.0/fastlane/lib/fastlane/lane_manager.rb:56:in `cruise_lane'
	 6: from /usr/local/lib/ruby/gems/2.6.0/gems/fastlane-2.135.0/fastlane/lib/fastlane/runner.rb:45:in `execute'
	 5: from /usr/local/lib/ruby/gems/2.6.0/gems/fastlane-2.135.0/fastlane/lib/fastlane/runner.rb:45:in `chdir'
	 4: from /usr/local/lib/ruby/gems/2.6.0/gems/fastlane-2.135.0/fastlane/lib/fastlane/runner.rb:49:in `block in execute'
	 3: from /usr/local/lib/ruby/gems/2.6.0/gems/fastlane-2.135.0/fastlane/lib/fastlane/lane.rb:33:in `call'
	 2: from Fastfile:107:in `block (2 levels) in parsing_binding'
	 1: from Fastfile:42:in `next_build_number'
Fastfile:42:in `+': [!] no implicit conversion of Integer into String (TypeError)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
  </pre>
</details>

### Environment

<details>
  <pre>
<details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                       |
| --------------------------- | ------------------------------------------- |
| OS                          | 10.13.6                                     |
| Ruby                        | 2.6.5                                       |
| Bundler?                    | false                                       |
| Git                         | git version 2.24.0                          |
| Installation Source         | /usr/local/bin/fastlane                     |
| Host                        | Mac OS X 10.13.6 (17G9016)                  |
| Ruby Lib Dir                | /usr/local/Cellar/ruby/2.6.5/lib            |
| OpenSSL Version             | OpenSSL 1.1.1d  10 Sep 2019                 |
| Is contained                | false                                       |
| Is homebrew                 | false                                       |
| Is installed via Fabric.app | false                                       |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/ |
| Xcode Version               | 10.1                                        |

### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   | en_US.UTF-8 | ✅ |
| LANGUAGE |             |   |

### fastlane files:

<details><summary>`./fastlane/Appfile`</summary>

```ruby
json_key_file("fastlane/secrets/google-play.json") # Path to the json secret file - Follow https://docs.fastlane.tools/actions/supply/#setup to get one
package_name("com.foo.bar") # e.g. com.krausefx.app

```
</details>

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.135.0 | ✅ Up-To-Date  |

### Loaded fastlane plugins:

| Plugin                             | Version | Update-Status |
| ---------------------------------- | ------- | ------------- |
| fastlane-plugin-versioning_android | 0.1.0   | ✅ Up-To-Date  |

<details><summary><b>Loaded gems</b></summary>

| Gem                                | Version      |
| ---------------------------------- | ------------ |
| did_you_mean                       | 1.3.0        |
| slack-notifier                     | 2.3.2        |
| atomos                             | 0.1.3        |
| CFPropertyList                     | 3.0.1        |
| claide                             | 1.0.3        |
| colored2                           | 3.1.2        |
| nanaimo                            | 0.2.6        |
| xcodeproj                          | 1.13.0       |
| rouge                              | 2.0.7        |
| xcpretty                           | 0.3.0        |
| terminal-notifier                  | 2.0.0        |
| unicode-display_width              | 1.6.0        |
| terminal-table                     | 1.8.0        |
| plist                              | 3.5.0        |
| public_suffix                      | 2.0.5        |
| addressable                        | 2.7.0        |
| multipart-post                     | 2.0.0        |
| word_wrap                          | 1.0.0        |
| tty-screen                         | 0.7.0        |
| tty-cursor                         | 0.7.0        |
| tty-spinner                        | 0.9.1        |
| babosa                             | 1.0.3        |
| colored                            | 1.2          |
| highline                           | 1.7.10       |
| commander-fastlane                 | 4.4.6        |
| excon                              | 0.68.0       |
| faraday                            | 0.17.0       |
| unf_ext                            | 0.0.7.6      |
| unf                                | 0.1.4        |
| domain_name                        | 0.5.20190701 |
| http-cookie                        | 1.0.3        |
| faraday-cookie_jar                 | 0.0.6        |
| faraday_middleware                 | 0.13.1       |
| fastimage                          | 2.1.7        |
| gh_inspector                       | 1.1.3        |
| json                               | 2.1.0        |
| mini_magick                        | 4.9.5        |
| multi_xml                          | 0.6.0        |
| rubyzip                            | 1.3.0        |
| security                           | 0.1.3        |
| xcpretty-travis-formatter          | 1.0.0        |
| dotenv                             | 2.7.5        |
| bundler                            | 1.17.2       |
| naturally                          | 2.2.0        |
| simctl                             | 1.6.6        |
| jwt                                | 2.1.0        |
| uber                               | 0.1.0        |
| declarative                        | 0.0.10       |
| declarative-option                 | 0.1.0        |
| representable                      | 3.0.4        |
| retriable                          | 3.1.2        |
| mime-types-data                    | 3.2019.1009  |
| mime-types                         | 3.3          |
| multi_json                         | 1.14.1       |
| signet                             | 0.12.0       |
| memoist                            | 0.16.0       |
| os                                 | 1.0.1        |
| googleauth                         | 0.6.7        |
| httpclient                         | 2.8.3        |
| google-api-client                  | 0.23.9       |
| google-cloud-env                   | 1.3.0        |
| google-cloud-core                  | 1.4.1        |
| digest-crc                         | 0.4.1        |
| google-cloud-storage               | 1.16.0       |
| emoji_regex                        | 1.0.1        |
| forwardable                        | 1.2.0        |
| logger                             | 1.3.0        |
| stringio                           | 0.0.2        |
| ipaddr                             | 1.2.2        |
| openssl                            | 2.1.2        |
| ostruct                            | 0.1.0        |
| strscan                            | 1.0.0        |
| date                               | 2.0.0        |
| fileutils                          | 1.1.0        |
| etc                                | 1.0.1        |
| io-console                         | 0.4.7        |
| zlib                               | 1.0.0        |
| rexml                              | 3.1.9        |
| psych                              | 3.1.0        |
| mutex_m                            | 0.1.0        |
| fastlane-plugin-versioning_android | 0.1.0        |
</details>

*generated on:* **2019-11-11**
</details>
 </pre>
</details>

**Comment by joshdholtz** [comment]
@edu-zamora On it!

**Comment by joshdholtz** [comment]
@edu-zamora This will be released in `2.135.1` in about 30 minutes or so

**Comment by edu-zamora** [comment]
@joshdholtz: This seems to work as expected and documented in `2.135.1`. Thank you a lot for the quick fix! :D
